jsonnet/components/kube-state-metrics.libsonnet

```diff
@@ -202,6 +231,8 @@ function(params)
                           --metric-denylist=
                           ^kube_secret_labels$,
                           ^kube_.+_annotations$
+                          ^kube_customresource_.+_annotations_info$,
+                          ^kube_customresource_.+_labels_info$,
```

Comment:
not needed?

Reply:
IIUC Both of these will be needed to filter out the CRS `*_info` and `*_label` metrics: https://github.com/openshift/cluster-monitoring-operator/pull/2078#discussion_r1417208635.